packages/react-composites/src/composites/CallWithChatComposite/CallWithChatControlBar.tsx

```diff
@@ -74,6 +75,12 @@ const inferCallWithChatControlOptions = (
  * @private
  */
 export const CallWithChatControlBar = (props: CallWithChatControlBarProps & ContainerRectProps): JSX.Element => {
+  useEffect(() => {
+    // On mount, button used for initial focus is hidden to prevent screen readers from announcing the initial focus
+    if (document.querySelector('[data-ui-id=call-with-chat-autofocus-hidden-button]') === document.activeElement) {
+      document.activeElement?.toggleAttribute('hidden');
```

Comment:
This reference to the `activeElement` should only show as this button when it has focus right? so the chain of events is:
1. user presses tab entering focus to the page
2. focus is on this button so the hook sets the CSS to hidden
3. without the user pressing tab again, because the current item was removed through CSS the focus goes to the microphone button